salt/modules/pkgng.py

```diff
@@ -475,7 +475,7 @@ def update_package_site(new_url):
     return True
 
 
-def stats(local=False, remote=False, jail=None, chroot=None, root=None):
+def stats(local=False, remote=False, bytes=False, jail=None, chroot=None, root=None):
```

Comment:
Can we add this kwarg at the end in case someone is calling this function with positional args?